csharp/src/Google.Protobuf/JsonTokenizer.cs

```diff
@@ -565,6 +572,38 @@ private void PopContainer()
                 };
             }
 
+            private static StringBuilder AcquireStringBuilder(int capacity = DEFAULT_CAPACITY)
```

Comment:
Remove the optional parameter as we're not using it?

Reply:
As we have private class with chance to make it internal - I would like to stay with it, but if you prefer I will submit new commit

Reply:
I see usages in project where it can be useful : [here](https://github.com/protocolbuffers/protobuf/blob/73769a030d127107be84e052b4afa3cbc62e77ee/csharp/src/Google.Protobuf/JsonParser.cs#L946C30-L946C31)